### PR TITLE
Fix typo in server_admin

### PR DESCRIPTION
**Pull request by RemiEven**

### CODE REVIEW

**Review by matthewhelmke — APPROVED** [review]
Thank you! From a language standpoint, the change is simple and easy to approve. @stianst, can you please have someone review from an engineering perspective?

File: server_admin/topics/admin-cli.adoc [hunk 1]
```diff
@@ -1084,7 +1084,7 @@ Use a dedicated [command]`add-roles` command to add realm roles to a user.
 Use the following example to add the [command]`user` role to user [command]`testuser`.
 [options="nowrap"]
 ----
-$ kcadm.sh add-roles --username testuser --rolename user -r demorealm
+$ kcadm.sh add-roles --uusername testuser --rolename user -r demorealm
```
Comment (stianst) [inline server_admin/topics/admin-cli.adoc:1087]: This doesn't look right. Surely the param name isn't "--uusername", but "--username" as it is in the docs currently

**Comment by RemiEven** [comment]
When I tried the commands without the double 'u', they failed. Moreover the help from the command line (kcadm.sh help add-roles) explicitly mentions "uusername" multiple times.

**Comment by stianst** [comment]
In that case I'd say the typo is in the CLI and not the docs. It makes no sense that the param is uusername

**Pull request merged**